metadata/metadata.go

```diff
@@ -85,8 +104,14 @@ func Filter(req *http.Request) (string, error) {
 		cleanedPath += "/"
 	}
 
+	for key := range req.URL.Query() {
+		if !knownQueryParameterKey[key] {
+			return "", fmt.Errorf("Unrecognized query parameter key: %#q.", key)
```

Comment:
Still has a period. Is that intentional?

Reply:
whoops, sorry, done